### PR TITLE
update: quarkus version

### PR DESCRIPTION
**Pull request by shivam-sharma7**
update: quarkus version

### CODE REVIEW

**Comment by jesperpedersen** [comment]
Conflict needs to be resolved

File: pom.xml [hunk 1]
```diff
@@ -76,8 +76,9 @@
         <graalvm.version>22.3.0</graalvm.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <quarkus.version>2.16.4.Final</quarkus.version>
-        <quinoa.version>1.2.7</quinoa.version>
+ 
+        <quarkus.version>2.16.5.Final</quarkus.version>
+        <quinoa.version>1.2.5</quinoa.version>
```
Comment (johnaohara) [inline pom.xml:81]: Why downgrade Quiona?

Comment (jesperpedersen) [reply pom.xml:81]: It is a wrong rebase; I'm sure that @shivam-sharma7 will fix it

Comment (shivam-sharma7) [reply pom.xml:81]: I think it's for `quarks 2.x` check release https://github.com/quarkiverse/quarkus-quinoa/releases

**Comment by jesperpedersen** [comment]
@shivam-sharma7 You reverted the Quinoa upgrade, and the pull request needs to be squashed

File: pom.xml (hunk 1 above)
Comment (johnaohara) [reply pom.xml:81]: ok, no problem

**Pull request closed without merging**